src/zui/ZUIMarkdown.tsx

```diff
@@ -6,10 +6,33 @@ import ZUICleanHtml from './ZUICleanHtml';
 interface ZUIMarkdownProps {
   BoxProps?: BoxProps;
   markdown: string;
+  forceTargetBlank?: boolean;
 }
 
-const ZUIMarkdown: React.FC<ZUIMarkdownProps> = ({ BoxProps, markdown }) => {
+const ZUIMarkdown: React.FC<ZUIMarkdownProps> = ({
+  BoxProps,
+  markdown,
+  forceTargetBlank,
+}) => {
   const dirtyHtml = marked(markdown, { breaks: true });
+  if (forceTargetBlank) {
+    const parser = new DOMParser();
+    const parsedHtml = parser.parseFromString(dirtyHtml, 'text/html');
+    const aNodes = parsedHtml.querySelectorAll('a');
+    aNodes.forEach((a) => {
+      a.setAttribute('target', '_blank');
+      a.setAttribute('rel', 'noopener noreferrer');
+    });
+    const serializer = new XMLSerializer();
+    const dirtyNewTabHtml = serializer.serializeToString(parsedHtml);
```

Comment:
This is an interesting solution. In my solution in #2803 I modified the DOM after rendering it, but you have instead opted to modify the HTML before submitting it to the DOM. What is the reason for that?